examples/cms-dotcms/.env.example

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_DOTCMS_HOST=""
+NEXT_PREVIEW_SECRET_TOKEN=""
+DOTCMS_PREVIEW_SECRET=""
```

Comment:
Because of these, let's call the file `.env.local.example` so it won't be accidentally committed once `.example` is removed.

Reply:
Yes! Great catch.